anesthetic/samples.py

```diff
@@ -611,6 +611,38 @@ def dlogX(self, nsamples=None):
         else:
             return WeightedDataFrame(dlogX, self.index, weights=self.weights)
 
+    def importance_reweighting(self, logL_new, replace=True, add=False):
```

Comment:
I chose to call the method `importance-reweighting`, because I fell like just `reweighting` insinuates that the `weights` are being accessed/changed, when really you are supposed to provide a new `logL`.

Reply:
see comment below about function signature

Reply:
> Would `importance_sample` resolve this?

Renamed accordingly in [bfd5183](https://github.com/williamjameshandley/anesthetic/pull/122/commits/bfd51830f904a35ac81984698231f2c620f68c92).